server/paymentProviders/stripe/index.js

```diff
@@ -261,8 +257,6 @@ export default {
       }
       if (event.type === 'invoice.payment_succeeded') {
         return creditcard.webhook(requestBody, event);
-      } else if (event.type === 'charge.refund.updated') {
-        return alipay.webhook(requestBody, event);
       } else if (event.type === 'source.chargeable') {
```

Comment:
While we're there, we can remove this.

Reply:
This was already removed in #8029. I'll rebase this after we merge that.